app/display_handler.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+from app.book import Base
+
+
+class DisplayHandler(Base, ABC):
+    @abstractmethod
+    def display(self) -> None:
+        pass
+
+
+class ConsoleDisplayHandler(DisplayHandler):
+    def display(self) -> None:
+        print(self.book.content)
+
+
+class ReverseDisplayHandler(DisplayHandler):
+    def display(self) -> None:
+        print(self.book.content[::-1])
```

Comment:
1 blank line before and after class methods/subclasses
Also it's better to name files in plural form